lightning/src/ln/channel.rs

```diff
@@ -7838,17 +7838,15 @@ impl<SP: Deref> FundedChannel<SP> where
 
 	fn internal_htlc_satisfies_config(
 		&self, htlc: &msgs::UpdateAddHTLC, amt_to_forward: u64, outgoing_cltv_value: u32, config: &ChannelConfig,
-	) -> Result<(), (&'static str, LocalHTLCFailureReason)> {
+	) -> Result<(), LocalHTLCFailureReason> {
```

Comment:
Is there more in `FundedChannel.send_htlc`?

Reply:
Seems like a shame to miss out on the dynamic values that are included in these messages (eg: Cannot send less than our next-HTLC minimum - {} msat).

The proper way to do this would be to surface the amount in the `LocalHTLCFailureReason`, but I don't think that's a worthwhile change just to delete a string. Would rather wait for the API to get some use and see whether that additional data would be valuable.

Reply:
I think leave as-is or go all the way to add the extra data. I'll make an issue about surfacing that info (so people can comment if it's valuable) + link this there.